src/Extension/CoreExtension.php

```diff
@@ -915,6 +916,20 @@ public static function keys($array): array
         return array_keys($array);
     }
 
+    /**
+     * Invokes a callable
+     *
+     * @param \Closure $arrow
+     *
+     * @internal
+     */
+    public static function invoke(Environment $env, $arrow, ...$arguments): mixed
+    {
+        self::checkArrow($env, $arrow, 'invoke', 'filter');
```

Comment:
Can be removed. It's used for BC on 3.x, but as this is a new method, we can directly type hint the param with `\Closure` like done in 4.x for the other existing methods.